docs/pages/docs/core-concepts/pipelines.mdx

```diff
@@ -203,7 +203,7 @@ The example below describes the `deploy` script of a `frontend` application that
 }
 ```
 
-This explicit configuration for `frontend#deploy` may seem to conflict with the `test` and `deploy` task configurations, but it does not. Since `test` and `deploy` do not have topological dependencies (e.g. `^<task>`), they can execute any time after their workspace's `build` and `test` scripts have finished.
+This explicit configuration for `frontend#deploy` may seem to conflict with the `test` and `deploy` task configurations, but it does not. Since `test` and `deploy` do not have dependencies in other workspaces (e.g. `^<task>`), they can execute any time after their workspace's `build` and `test` scripts have finished.
```

Comment:
```suggestion
This explicit configuration for `frontend#deploy` may seem to conflict with the `test` and `deploy` task configurations, but it does not. Since `test` and `deploy` do not have dependencies on other workspaces (e.g. `^<task>`), they can execute any time after their workspace's `build` and `test` scripts have finished.
```